python/semantic_kernel/core_skills/planning/plan.py

```diff
@@ -0,0 +1,17 @@
+from typing import Any
+
+
+class Plan:
```

Comment:
See the latest `Plan` object in dotnet.

https://github.com/microsoft/semantic-kernel/pull/389/files#diff-128f2b23a8593a275761d05e910b08ee6e369f7323487d1d13aed49cdc3e3dbf